django_sharding_library/decorators.py

```diff
@@ -38,5 +39,24 @@ def configure(cls):
             setattr(cls, 'django_sharding__shard_group', shard_group)
             setattr(cls, 'django_sharding__is_sharded', True)
 
+            # If the sharded by field is set, we will make our custom manager the default manager.
+            # todo: update docs to indicate that ShardManager should be used as the base class for any custom managers->
+            # for a model that defines a sharded by field
+            # todo: Update docs to include the sharded by field and the get_shard_from_id function
+            if sharded_by_field:
+                from django.db.models import Manager
+                if not isinstance(cls.objects, ShardManager):
+                    if type(cls.objects) == Manager:
```

Comment:
This is alright for now and possibly for good, but we should investigate using `contribute_to_class` in order to add functionality to the inherited class instead of making you do the inheritance. I've not worked with that API before so it may not work out, but we can explore it or cross it out as an option later.